src/app/(protected)/notebooks/[notebook_id]/[page_id]/page.tsx

```diff
@@ -1,17 +1,21 @@
 "use client";
 
-import { Sidebar } from "@/components/notebooks/sidebar";
+import { PagesNavigation } from "@/components/notebooks/pages-navigation";
 import { Loader } from "@/components/loader";
 import { useQuery, gql } from "@apollo/client";
+import _ from 'lodash';
 
 const NOTEBOOK_QUERY = gql`
   query GetNotebook($id: ID!) {
     notebook(id: $id) {
       id
       title
+      ownerId
```

Comment:
We don't need to query `ownerId` and `ownerType`


Part of the beauty of using GraphQL. We can query only the data we need.